src/App.tsx

```diff
@@ -15,189 +50,61 @@ export const App: React.FC = () => {
 
       <div className="todoapp__content">
         <header className="todoapp__header">
-          {/* this button should have `active` class only if all todos are completed */}
-          <button
-            type="button"
-            className="todoapp__toggle-all active"
-            data-cy="ToggleAllButton"
-          />
+          {todos.length > 0 && (
+            <button
+              type="button"
+              className={cn('todoapp__toggle-all', {
+                active: todos.length > 0 && todos.every(todo => todo.completed),
```

Comment:
Create variable for that `todos.length > 0 && todos.every(todo => todo.completed)`